plugins/woocommerce-blocks/docs/third-party-developers/extensibility/hooks/migrated-hooks.md

```diff
@@ -1,3 +1,9 @@
+---
+post_title: Cart and Checkout Filters - Legacy Hooks and Filters
```

Comment:
```suggestion
post_title: Cart and Checkout - Legacy hooks
```

Reply:
I'd either use `Legacy hooks` or `Legacy actions and filters`, as `actions` and `filters` are `hooks`.